content/200-concepts/100-components/01-prisma-schema/index.mdx

```diff
@@ -8,8 +8,8 @@ metaDescription: 'The Prisma schema is the main configuration file when using Pr
 
 The Prisma schema file (short: _schema file_, _Prisma schema_ or _schema_) is the main configuration file for your Prisma setup. It is typically called `schema.prisma` and consists of the following parts:
 
-- [**Data sources**](data-sources): Specify the details of the data sources Prisma should connect to (e.g. a PostgreSQL database)
-- [**Generators**](generators): Specifies what clients should be generated based on the data model (e.g. Prisma Client)
+- [**Data sources**](https://www.prisma.io/docs/concepts/components/prisma-schema/data-sources): Specify the details of the data sources Prisma should connect to (e.g. a PostgreSQL database)
+- [**Generators**](https://www.prisma.io/docs/concepts/components/prisma-schema/generators): Specifies what clients should be generated based on the data model (e.g. Prisma Client)
```

Comment:
Here I can see the difference between the links that are changed, and the ones that are not is that line 13 is using the proper "fully qualified" link that we want to use in the docs vs. lines 11 and 12 do not yet.

If the link in line 13 is fine right now, the fix here (and probably for most other suggested changes) could be:

```suggestion
- [**Data sources**](/concepts/components/prisma-schema/data-sources): Specify the details of the data sources Prisma should connect to (e.g. a PostgreSQL database)
- [**Generators**](/concepts/components/prisma-schema/generators): Specifies what clients should be generated based on the data model (e.g. Prisma Client)
```